docs/UserGuide.md

```diff
@@ -301,6 +312,11 @@ This is an intentional design choice to ensure that the command is used for targ
 - Month input should be a valid month in `MMM` format and non-empty.
   - Filtering by month "September" should be `filter b/Sep`
     - `filter b/SEP` matches person with Birthday in September.
+- Month input will also accept extraneous input,
+  as long as it is not the same prefix as that will be considered as duplicate prefixes.
+  If the first few characters are any valid month, it will be valid. 
+  E.g.,
+  This forgiving behavior allows you to type in `filter b/AprMogger` and Realodex will interpret the month as April!
```

Comment:
```suggestion
- Month input will also accept extraneous input, i.e. if the first 3 characters are any valid month, it will be valid. 
    E.g., `filter b/AprMogger` will result in filtering by the month April
  ⚠️ This does not apply if the extraneous input is the **same prefix** as that will be considered as **duplicate prefixes.** (e.g. `filter b/may b/`)
```

hmm it seems like filter `b/mayb/` works but not `b/may b/` should we explicitly say it?

Reply:
okiess

Reply:
Fixed